### ISSUE TITLE
Wrong alignment when loading the User page directly after informing OTP

### ISSUE THREAD

**Issue by eduardomozart** [nilsteampassnet/TeamPass#3428]
### Steps to reproduce
1. After generating the OTP for a local TeamPass user, login with the user.
2. Access the Users page directly e.g. https://teampass.example.com/index.php?page=users
3. Login into TeamPass.
4. Inform the OTP code.
5. The Users page is loaded but it's with a strange alignment

### Expected behaviour

Reloading the page makes the Users page loading as expected.

### Actual behaviour

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/2974895/204043162-821f2e56-aac3-41c2-a271-1aa3d673b428.png">

### Server configuration
**Operating system**: Linux cloud.ifantasy.com.br 3.10.0-962.3.2.lve1.5.70.el7.x86_64 #1 SMP Mon May 30 08:45:55 UTC 2022 x86_64

**Web server:** Apache

**Database:** 5.7.30 - MySQL Community Server (GPL)

**PHP version:** 7.4.30

**Teampass version:** 3.0.0.21

**Teampass configuration file:** [tp.config.php.txt](https://github.com/nilsteampassnet/TeamPass/files/10087406/tp.config.php.txt)

**Updated from an older Teampass or fresh install:** Updated from 2.1.27.36 to 3.0.0.21
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:** Chrome 107.0.5304.110

**Operating system:** macOS 12.6.1

#### Web server error log

No errors being reported at Apache error log.

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by nilsteampassnet** [comment]
I believe this is more browser related.
Perform a CTRL+F5 after each upgrade could help to clear the local cache.

**Comment by eduardomozart** [comment]
Yes, it was already refreshed and opens just fine when I refresh the page. It opens just fine after user already inputs the OTP before. It renders incorrectly only when trying to access the Users page directly using a user that haven’t input the OTP yet and attempt to load the user page directly. Maybe forcing the browser to reload the page into this specific kind of situation would be a solution for this problem, as reloading the page it works as expected.

**Comment by nilsteampassnet** [comment]
Closed automatically due to inactivity (no response for over 1 year).
